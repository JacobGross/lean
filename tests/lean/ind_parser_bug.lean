import logic

inductive acc1 (A : Type) (R : A → A → Prop) (a : A) :  Prop :=
intro : ∀ (x : A), (∀ (y : A), R y x → acc1 A R y) → acc1 A R x

section

  variables (A : Type) (R : A → A → Prop)

  inductive acc2 (a : A) : Prop :=
  intro : ∀ (x : A), (∀ (y : A), R y x → acc2 y) → acc2 x

end

print "done"
