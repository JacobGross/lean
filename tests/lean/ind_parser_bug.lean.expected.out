ind_parser_bug.lean:4:39: error: invalid datatype declaration, mismatch in the #3 explicit parameter
ind_parser_bug.lean:11:41: error: invalid datatype declaration, mismatch in the #1 explicit parameter
done
